src/elm/Session/LoggedIn.elm

```diff
@@ -191,6 +206,25 @@ type ProfileStatus
     | Loaded Profile.Model
 
 
+type alias AddContactModal =
+    { show : Bool
+    , contactOption : String
+    , country : String
+    , contact : String
+    , contactProblems : Maybe (List String)
+    }
+
+
+initAddPhoneModal : Bool -> AddContactModal
+initAddPhoneModal show =
```

Comment:
I guess the name should be `initAddContactModal`.

Reply:
All right, will work on that, thanks! I was initially thinking that the user could only add a phone number, so I must've forgotten to rename some stuff.